wordpress-mobile/WordPress-iOS
# Unified about screen: Add footer view

Implements the Automattic logo footer view for the screen.

<img src="https://user-images.githubusercontent.com/4780/139885179-5a6a2d8b-043d-470a-a292-7d752c9aefbe.png" width="350">

**To test**

- Build and run, check the footer looks as expected
- Check that it changes color in dark mode

## Regression Notes
1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
